multichain-testing/scripts/make-bank-asset-info.ts

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env tsx
+/* eslint-env node */
+
+import '@endo/init';
+import starshipChainInfo from '../starship-chain-info.js';
+import { makeAssetInfo } from '../tools/asset-info.ts';
+
+const main = () => {
+  if (!starshipChainInfo) {
+    throw new Error(
+      'starshipChainInfo not found. run `make override-chain-registry` or `node_modules/.bin/tsx scripts/fetch-starship-chain-info.ts` first.',
```

Comment:
the `node_modules/.bin/` again jumps out at me... it looks like we're peeking into the internals of something.

```suggestion
      'starshipChainInfo not found. run `make override-chain-registry` or `./scripts/fetch-starship-chain-info.ts` first.',
```